docs/content/aggregates.rst

```diff
@@ -6,21 +6,34 @@ Aggregates
 
 Aggregates are a key-value store specific to an account.
 Each time a new aggregate message is received for a specific account, the
-nodes add a new layer on top and mutate the global storage for this account.
+nodes update the aggregate for this account.
+
+Like a dictionary update, if a key already exists, it is updated,
+otherwise it is created.
 
 Query aggregate of an account
 -----------------------------
 
-To query keys from an account aggregate, you need to call the fetch
-aggregate function.
+To query keys from an account aggregate, you need to call the
+fetch_aggregate function on the client.
+
+Since version 0.8.0, only the asynchronous methods are available.
+To use them in the Python REPL, simply start it with:
+
+.. code-block:: bash
+
+    $ python3 -m asyncio
 
-Synchronous version:
+Then you can use the asynchronous methods:
 
 .. code-block:: python3
```

Comment:
Do you need to use `python -m asyncio` to have access to `await` in the REPL ? If so, should that be documented ? Or the altenative using `asyncio.run(coroutine)` ?

Reply:
Yeah, that's the case, I documented it just above

Reply:
Because the docs were centered around REPL use, I thought I just keep it that way. If people want to know more about async, they can look it up themselves.